smoke-tests/src/test/java/com/splunk/opentelemetry/SpringBootSmokeTest.java

```diff
@@ -25,18 +28,22 @@
 
 class SpringBootSmokeTest extends SmokeTest {
 
-  private String getTargetImage(int jdk) {
-    return "open-telemetry-docker-dev.bintray.io/java/smoke-springboot-jdk"
-        + jdk
-        + ":20201105.347264626";
+  private TestImage getTargetImage(int jdk) {
+    return linuxImage(
+        "open-telemetry-docker-dev.bintray.io/java/smoke-springboot-jdk"
+            + jdk
+            + ":20201105.347264626");
   }
 
   @ParameterizedTest(name = "{index} => SpringBoot SmokeTest On JDK{0}.")
   @ValueSource(ints = {8, 11, 15})
   public void springBootSmokeTestOnJDK(int jdk) throws IOException, InterruptedException {
-    startTarget(getTargetImage(jdk));
+    TestImage image = getTargetImage(jdk);
+
+    startTargetOrSkipTest(image);
 
-    String url = String.format("http://localhost:%d/greeting", target.getMappedPort(8080));
+    String url =
+        String.format("http://localhost:%d/greeting", containerManager.getTargetMappedPort(8080));
```

Comment:
This can now use `AppServerTest.getUrl`, right?

Reply:
Done, had to make it extend `AppServerTest` first though.